src/Razor/src/Microsoft.VisualStudio.LanguageServices.Razor/ProjectSystem/DefaultRazorProjectHost.cs

```diff
@@ -83,7 +83,7 @@ protected override async Task DisposeCoreAsync(bool initialized)
         {
             await base.DisposeCoreAsync(initialized).ConfigureAwait(false);
 
-            if (initialized)
+            if (initialized && _subscription != null)
```

Comment:
During debugging I ran into a scenario where this happened. I know it was probably because of a temporary mistake on my part, but I figured it made sense to be defensive rather than throw from a Dispose method.